.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+
+language: bash
+
```

Comment:
jeste by se melo nejak zaridit, aby se to testovalo v PHP 7.1 i 7.2, tohle nemusi bezet vzdycky v danym image

Reply:
Super poznámka! Zkouším to zkombinovat.